packages/core/EventManager.ts

```diff
@@ -388,8 +389,7 @@ export default class EventManager {
 
         log.debug("RescheduleOrganizerChanged: Creating Event and Meeting for for new booking");
 
-        const newEvent = { ...evt, destinationCalendar: newDestinationCalendar };
-        const createdEvent = await this.create(newEvent);
+        const createdEvent = await this.create(originalEvt);
```

Comment:
`create` function updates the videoCallData in evt(Calendar event object) like 
evt.videoCallData = { ... }

but in reschedule function we passed copy of event object to create in this line which updated the copy of event but not the original event